src/containers/blogs/Blogs.js

```diff
@@ -24,20 +61,34 @@ export default function Blogs() {
         </div>
         <div className="blog-main-div">
           <div className="blog-text-div">
-            {blogSection.blogs.map((blog, i) => {
-              return (
-                <BlogCard
-                  key={i}
-                  isDark={isDark}
-                  blog={{
-                    url: blog.url,
-                    image: blog.image,
-                    title: blog.title,
-                    description: blog.description
-                  }}
-                />
-              );
-            })}
+            {blogSection.displayMediumBlogs !== "true"
+              ? blogSection.blogs.map((blog, i) => {
+                  return (
+                    <BlogCard
+                      key={i}
+                      isDark={isDark}
+                      blog={{
+                        url: blog.url,
+                        image: blog.image,
+                        title: blog.title,
+                        description: blog.description
+                      }}
+                    />
+                  );
+                })
+              : mediumBlogs.map((blog, i) => {
```

Comment:
Seems like this doesn't work and broke the main site. from console:
```
TypeError: x.map is not a function
    H Blogs.js:79
    React 7
    unstable_runWithPriority scheduler.production.min.js:19
    React 5
    y Blogs.js:11
    H Blogs.js:39
    promise callback*H/< Blogs.js:35
    React 2
    unstable_runWithPriority scheduler.production.min.js:19
    React 4
    unstable_runWithPriority scheduler.production.min.js:19
    React 6
    <anonymous> index.js:7
    i (index):1
    <anonymous> main.99d3a735.chunk.js:1
    i (index):1
    r (index):1
    t (index):1
    <anonymous> main.99d3a735.chunk.js:1
react-dom.production.min.js:209:194
```

Reply:
displayMediumBlogs needs to be set to false in portofolio.js, I've just commited this after merge

Reply:
https://github.com/saadpasta/developerFolio/pull/446 - Created additional pull request